README.md

```diff
@@ -168,18 +170,87 @@ If a new certificate is required, create a new keystore with a certificate by ru
+   
+   keystore-type: type of certificate keystore being used (if generated through our script use "pkcs12")
+   
+   password-file: path to the file with the password used for accessing the certificate keystore (if generated through our script they are by default located in "./tls/password")
 2. Enable TLS in specific config:
-    - For JSON RPC: `mantis.network.rpc.http.mode=https`
-3. Configure the certificate used from RpcClient to connect with the node. Necessary if the node uses http secure. 
+   - For JSON RPC: `mantis.network.rpc.http.mode=https`
+3. Configure the certificate used from RpcClient to connect with the node. Necessary if the node uses http secure.
    This certificate and password file to be used at `faucet.rpc-client.certificate` key on the `faucet.conf` file:
 
-    keystore-path: path to the keystore storing the certificates
-    keystore-type: type of certificate keystore being used (if generated through our script use "pkcs12")
-    password-file: path to the file with the password used for accessing the certificate keystore
+   keystore-path: path to the keystore storing the certificates
+   keystore-type: type of certificate keystore being used (if generated through our script use "pkcs12")
+   password-file: path to the file with the password used for accessing the certificate keystore
+
+### Faucet setup and testing
+1. First start a client node using the docker-compose, by running the script found at `./docker/mantis/build.sh`
+Modify the script before running it by adding the `volumes` and `command` sections to mantis configuration:
+```
+mantis:
+image: mantis:latest
+ports:
+- 8546:8546
+- 13798:13798
+- 9095:9095
+networks:
+- mantis-net
+volumes:
+- $HOME/.mantis:/home/demiourgos728/.mantis/
+command: -Dconfig.file=./conf/sagano.conf
+```
+   
+2. Create a wallet address. Run the following curl command, replacing `<password>` by a password of your choice:
+```
+curl --request POST \
+  --url http://127.0.0.1:8546/ \
+  --header 'Cache-Control: no-cache' \
+  --header 'Content-Type: application/json' \
+  --data '{
+	"jsonrpc": "2.0",
+  "method": "personal_newAccount", 
+  "params": ["<password>"],
+  "id": 1
+}'
+```
+
+You will receive a response like this:
+```
+{"jsonrpc":"2.0","result":"<address>","id":1}
+```
+
+3. Modify `src/universal/conf/faucet.conf` file, config your account address created in the previous step. with the password choosen by you:
+```
+wallet-address = "<address>"
+wallet-password = "<password>"
+```
+
+4. Now check the `keystore` folder in `~/.mantis/testnet-internal-nomad/keystore`.
+Inside you will find a key generate with the curl request sent in step `2.`. Copy that file to `~/.mantis-faucet/keystore/`:
+```
+cp UTC--<date>--<key> ~/.mantis-faucet/keystore/
+```
+
+5. Start the faucet in command line:
+```
+sbt -Dconfig.file=src/universal/conf/faucet.conf "run faucet"
+```
+
+6. Run the following curl command to send tokens from your faucet to a wallet address:
```

Comment:
A newly created faucet won't have any ETC to send. Maybe add a note to avoid confusion. Something like `Note that in order to send ETC the faucet needs sufficient funds in its account.`

Reply:
You are right, I forgot to add that detail. You can check again